src/components/deposit/Congratulations.js

```diff
@@ -27,22 +36,25 @@ const Congratulations = ({ depositAddress, lotInSatoshis, signerFeeInSatoshis })
       </div>
       <hr />
       <div className="description">
+        {
+          depositAddress && depositAddress.length > 0
+          ? <>
+              <div className="deposit-address-label">Deposit Address:</div>
+              <CopyAddressField address={depositAddress} />
+            </>
+          : ''
```

Comment:
I see we were doing this `depositAddress && depositAddress.length > 0` check before as well… I think we may want to just drop it, honestly. If you land on this page and `depositAddress` is missing, something is very wrong, right?

Again, no real block here.

Reply:
Will clean it up - the check was pre-existing logic!

Reply:
Addressed in 940d94b